packages/qwik/src/optimizer/src/plugins/rollup.ts

```diff
@@ -78,6 +78,7 @@ export function qwikRollup(qwikRollupOpts: QwikRollupPluginOptions = {}): any {
 
     outputOptions(rollupOutputOpts) {
       return normalizeRollupOutputOptionsObject(
+        qwikPlugin.getOptimizer(),
```

Comment:
We have 3 `qwikPlugin.smth` arguments here. I'll make a follow-up PR so that we only pass a `qwinPlugin` as an argument.